imagebuildah/stage_executor.go

```diff
@@ -1799,6 +1799,21 @@ func (s *StageExecutor) intermediateImageExists(ctx context.Context, currNode *p
 		}
 	}
 	for _, image := range images {
+		// If `--cache-ttl` was specified then
+		// then ignore processing image if it
+		// was created before the specified
+		// duration.
```

Comment:
We're in the library here.  The application that's using the library may not have an option with this name.

Reply:
amended.